pipelines/open_edx/codejail_docker_build_pipeline.yaml

```diff
@@ -0,0 +1,67 @@
+---
+
+resources:
+- check_every: 24h
+  name: codejail-release
+  type: github-release
+  source:
+    owner: eduNEXT
+    repository: codejailservice
+    branch: main
+    release: true
+
+# - check_every: 24h
+#   name: codejail-release
+#   type: git
+#   source:
+#     uri: https://github.com/openedx/cs_comments_service.git
+#     branch: ((openedx_branch))
+
+- name: codejail-image
+  type: registry-image
+  icon: docker
+  source:
+    repository: mitodl/codejail
+    tag: $(cat ./codejail-release/tag)
+    # username: mitxdevops
+    # password: eDArTnGpkOdnNTnCoFpvnddCLfTjAn
```

Comment:
These lines shouldn't be present in this file. If this is the actual credentials for our account then we need to rotate them now.